src/main/java/io/vertx/mqtt/impl/MqttClientImpl.java

```diff
@@ -1061,4 +1180,18 @@ private boolean isValidStringSizeInUTF8(String string) {
 
     return false;
   }
+
+  private class ExpiringPacket {
```

Comment:
Can you add a little bit of Javadoc for the class, please?

Reply:
@sophokles73 ping

Reply:
will do